clouddriver-aws/src/main/groovy/com/netflix/spinnaker/clouddriver/aws/deploy/handlers/BasicAmazonDeployHandler.groovy

```diff
@@ -107,9 +107,8 @@ class BasicAmazonDeployHandler implements DeployHandler<BasicAmazonDeployDescrip
         }
       }
 
-      def blockDeviceMappingForInstanceType = BlockDeviceConfig.blockDevicesByInstanceType[description.instanceType]
-      if (blockDeviceMappingForInstanceType) {
-        description.blockDevices = blockDeviceMappingForInstanceType
+      if (!description.blockDevices) {
```

Comment:
Does this allow for an explicitly-empty set of block devices (not an aws expert, but wouldn't it be a reasonable use-case for one to request a server group whose instances won't be associated with any additional block devices)?

If this is a reasonable use-case, this should probably be `if (description.blockDevices == null)`.

If not reasonable, feel free to disregard this comment.